.github/workflows/test.yml

```diff
@@ -17,8 +17,10 @@ jobs:
       matrix:
           os: [ubuntu-18.04]
     steps:
-    - uses: ros-tooling/setup-ros@0.0.13
-    - uses: ros-tooling/action-ros-ci@0.0.13
+    - uses: ros-tooling/setup-ros@0.0.22
+    # workaround to ensure ament_package dependency is installed
+    - run: python3 -m pip install -U --force-reinstall importlib-resources
```

Comment:
On bionic, you are actually also going to need to pip install `importlib-metadata` as well.

Reply:
Hm, interesting that tests passed. I wonder if that was already pulled in as a dependency.

Reply:
> Hm, interesting that tests passed. I wonder if that was already pulled in as a dependency.

Yeah, that's possible.  It's also possible that you just haven't run into the pieces of code that require it yet.  Nevertheless, best to have them installed, especially as there is still some work in-flight around this that will hopefully land soon: https://github.com/ros2/ros2/issues/919

Another way to fix this is to switch to Focal, but that may be a bridge too far right now.